spec/features/schools/participants/add_participants/sit_adding_self_as_mentor/happy_path_sit_adds_self_as_mentor_spec.rb

```diff
@@ -17,7 +17,7 @@
     set_sit_dqt_validation_result
   end
 
-  scenario "Induction tutor can add themselves as a mentor and validates" do
+  xscenario "Induction tutor can add themselves as a mentor and validates" do
```

Comment:
This test was skipped because in the new dashboard the SITs still has no way to add themselves as mentors.
This is something I need to speak with Nicola.